scrapy/core/scheduler.py

```diff
@@ -13,9 +12,11 @@
 
 class Scheduler(object):
 
-    def __init__(self, dupefilter, jobdir=None, dqclass=None, mqclass=None, logunser=False, stats=None):
+    def __init__(self, dupefilter, jobdir=None, logunser=False, stats=None,
+                 pqclass=None, dqclass=None, mqclass=None):
```

Comment:
New argument order is more clean, but changing the order of arguments is technically backwards-incompatible.